docs/resources/kernel_module.md.erb

```diff
@@ -39,6 +40,12 @@ where
 
 The following examples show how to use this InSpec audit resource.
 
+### version
+
+The `version` property tests if the named module version is on the system:
+
+    its(:version) { should eq '3.2.2' }
```

Comment:
Do we have other resources with symbol properties? I confirmed 'version' works as well.